http://www.nntp.perl.org/group/perl.perl5.changes/2008/04/msg21478.html
- minus the change in Module::CoreList that we are upgrading

--- perl/Porting/Maintainers.pm#2~33194~	2008-02-02 09:05:25.000000000 -0800
+++ perl/Porting/Maintainers.pm	2008-04-03 09:03:24.000000000 -0700
@@ -14,11 +14,12 @@
 require "Maintainers.pl";
 use vars qw(%Modules %Maintainers);
 
-use vars qw(@ISA @EXPORT_OK);
+use vars qw(@ISA @EXPORT_OK $VERSION);
 @ISA = qw(Exporter);
 @EXPORT_OK = qw(%Modules %Maintainers
 		get_module_files get_module_pat
 		show_results process_options);
+$VERSION = 0.02;
 require Exporter;
 
 use File::Find;
@@ -107,9 +108,12 @@
     my @Files;
    
     if ($Opened) {
-	my @raw = `p4 opened`;
+	@Files = `p4 opened`;
 	die if $?;
-	@Files =  map {s!#.*!!s; s!^//depot/.*?/perl/!!; $_} @raw;
+	foreach (@Files) {
+	    s!#.*!!s;
+	    s!^//depot/(?:perl|.*?/perl)/!!;
+	}
     } else {
 	@Files = @ARGV;
     }

--- perl/ext/B/B/Concise.pm#2~33126~	2008-01-30 07:26:23.000000000 -0800
+++ perl/ext/B/B/Concise.pm	2008-04-03 09:03:24.000000000 -0700
@@ -14,7 +14,7 @@
 
 use Exporter (); # use #5
 
-our $VERSION   = "0.74";
+our $VERSION   = "0.75";
 our @ISA       = qw(Exporter);
 our @EXPORT_OK = qw( set_style set_style_standard add_callback
 		     concise_subref concise_cv concise_main

--- perl/ext/Devel/DProf/DProf.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/ext/Devel/DProf/DProf.pm	2008-04-03 09:03:24.000000000 -0700
@@ -230,7 +230,7 @@
 
 use XSLoader ();
 
-$Devel::DProf::VERSION = '20050603.00';  # this version not authorized by
+$Devel::DProf::VERSION = '20080331.00';  # this version not authorized by
 				         # Dean Roehrich. See "Changes" file.
 
 XSLoader::load 'Devel::DProf', $Devel::DProf::VERSION;

--- perl/ext/Devel/Peek/Peek.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/ext/Devel/Peek/Peek.pm	2008-04-03 09:03:24.000000000 -0700
@@ -3,7 +3,7 @@
 
 package Devel::Peek;
 
-$VERSION = '1.03';
+$VERSION = '1.04';
 $XS_VERSION = $VERSION;
 $VERSION = eval $VERSION;
 

--- perl/ext/POSIX/POSIX.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/ext/POSIX/POSIX.pm	2008-04-03 09:03:24.000000000 -0700
@@ -4,7 +4,7 @@
 
 our(@ISA, %EXPORT_TAGS, @EXPORT_OK, @EXPORT, $AUTOLOAD, %SIGRT) = ();
 
-our $VERSION = "1.13";
+our $VERSION = "1.14";
 
 use AutoLoader;
 

--- perl/ext/PerlIO/encoding/encoding.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/ext/PerlIO/encoding/encoding.pm	2008-04-03 09:03:24.000000000 -0700
@@ -1,7 +1,7 @@
 package PerlIO::encoding;
 
 use strict;
-our $VERSION = '0.10';
+our $VERSION = '0.11';
 our $DEBUG = 0;
 $DEBUG and warn __PACKAGE__, " called by ", join(", ", caller), "\n";
 

--- perl/ext/PerlIO/scalar/scalar.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/ext/PerlIO/scalar/scalar.pm	2008-04-03 09:03:24.000000000 -0700
@@ -1,5 +1,5 @@
 package PerlIO::scalar;
-our $VERSION = '0.05';
+our $VERSION = '0.06';
 use XSLoader ();
 XSLoader::load 'PerlIO::scalar';
 1;

--- perl/ext/PerlIO/via/via.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/ext/PerlIO/via/via.pm	2008-04-03 09:03:24.000000000 -0700
@@ -1,5 +1,5 @@
 package PerlIO::via;
-our $VERSION = '0.04';
+our $VERSION = '0.05';
 use XSLoader ();
 XSLoader::load 'PerlIO::via';
 1;

--- perl/ext/Socket/Socket.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/ext/Socket/Socket.pm	2008-04-03 09:03:24.000000000 -0700
@@ -1,7 +1,7 @@
 package Socket;
 
 our($VERSION, @ISA, @EXPORT, @EXPORT_OK, %EXPORT_TAGS);
-$VERSION = "1.80";
+$VERSION = "1.81";
 
 =head1 NAME
 

--- perl/lib/ExtUtils/Embed.pm#2~33574~	2008-03-26 09:37:45.000000000 -0700
+++ perl/lib/ExtUtils/Embed.pm	2008-04-03 09:03:24.000000000 -0700
@@ -19,7 +19,7 @@
 use strict;
 
 # This is not a dual-life module, so no need for development version numbers
-$VERSION = '1.27';
+$VERSION = '1.28';
 
 @ISA = qw(Exporter);
 @EXPORT = qw(&xsinit &ldopts 

--- perl/lib/Fatal.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/lib/Fatal.pm	2008-04-03 09:03:24.000000000 -0700
@@ -5,7 +5,7 @@
 use strict;
 our($AUTOLOAD, $Debug, $VERSION);
 
-$VERSION = 1.05;
+$VERSION = 1.06;
 
 $Debug = 0 unless defined $Debug;
 
@@ -106,7 +106,7 @@
         $proto = eval { prototype "CORE::$name" };
 	die "$name is neither a builtin, nor a Perl subroutine" 
 	  if $@;
-	die "Cannot make a non-overridable builtin fatal"
+	die "Cannot make the non-overridable builtin $name fatal"
 	  if not defined $proto;
 	$core = 1;
 	$call = "CORE::$name";

--- perl/lib/Fatal.t#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/lib/Fatal.t	2008-04-03 09:03:24.000000000 -0700
@@ -3,7 +3,7 @@
 BEGIN {
    chdir 't' if -d 't';
    @INC = '../lib';
-   print "1..15\n";
+   print "1..16\n";
 }
 
 use strict;
@@ -34,3 +34,9 @@
 eval { my $a = opendir FOO, 'lkjqweriuapofukndajsdlfjnvcvn' };
 print "not " if $@ =~ /^Can't open/;
 print "ok $i\n"; ++$i;
+
+eval { Fatal->import(qw(print)) };
+if ($@ !~ m{Cannot make the non-overridable builtin print fatal}) {
+    print "not ";
+}
+print "ok $i\n"; ++$i;

--- perl/lib/File/Basename.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/lib/File/Basename.pm	2008-04-03 09:03:24.000000000 -0700
@@ -54,7 +54,7 @@
 require Exporter;
 @ISA = qw(Exporter);
 @EXPORT = qw(fileparse fileparse_set_fstype basename dirname);
-$VERSION = "2.76";
+$VERSION = "2.77";
 
 fileparse_set_fstype($^O);
 

--- perl/lib/File/Find.pm#2~33162~	2008-01-31 14:54:31.000000000 -0800
+++ perl/lib/File/Find.pm	2008-04-03 09:03:24.000000000 -0700
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 use warnings::register;
-our $VERSION = '1.12';
+our $VERSION = '1.13';
 require Exporter;
 require Cwd;
 

--- perl/os2/OS2/REXX/REXX.pm#1~32694~	2007-12-22 01:23:09.000000000 -0800
+++ perl/os2/OS2/REXX/REXX.pm	2008-04-03 09:03:24.000000000 -0700
@@ -11,7 +11,7 @@
 # Other items we are prepared to export if requested
 @EXPORT_OK = qw(drop register);
 
-$VERSION = '1.03';
+$VERSION = '1.04';
 
 # We cannot just put OS2::DLL in @ISA, since some scripts would use
 # function interface, not method interface...

